Lombiq.OrchardCoreApiClient/ApiClientBase.cs

```diff
@@ -0,0 +1,39 @@
+using Lombiq.HelpfulLibraries.Refit.Helpers;
+using Lombiq.OrchardCoreApiClient.Models;
+using System;
+using System.Net.Http;
+
+namespace Lombiq.OrchardCoreApiClient;
+
+public class ApiClientBase<T> : IDisposable
```

Comment:
The non-generic `ApiClient` should remain, that's still useful as a default.

Reply:
...still named `ApiClient`.

Reply:
What do you mean?

Reply:
It is named `ApiClient`

Reply:
The non-generic client is named `OrchardCoreApiClient`, it should be `ApiClient` (since this is a usual pattern and also would prevent adding a breaking change).